core/server/proxy/src/main/java/alluxio/proxy/s3/ListBucketResult.java

```diff
@@ -11,56 +11,49 @@
 
 package alluxio.proxy.s3;
 
-import alluxio.AlluxioURI;
 import alluxio.client.file.URIStatus;
-import alluxio.wire.FileInfo;
 
-import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 /**
- * Get bucket result defined in http://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html
+ * Get bucket result defined in https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html
  * It will be encoded into an XML string to be returned as a response for the REST call.
- *
- * TODO(chaomin): consider add more required fields in S3 BucketGet API.
  */
 @JacksonXmlRootElement(localName = "ListBucketResult")
-@JsonPropertyOrder({ "Name", "Prefix", "ContinuationToken", "NextContinuationToken",
-    "KeyCount", "MaxKeys", "IsTruncated", "Contents" })
 public class ListBucketResult {
+  private static final Logger LOG = LoggerFactory.getLogger(ListBucketResult.class);
+
+  private static final int DEFAULT_MAX_KEYS = 1000;
 
   /* Name of the bucket. */
-  private String mName = "";
-  /* Keys that begin with the indicated prefix. */
-  private String mPrefix = null;
-  /* ContinuationToken is included in the response if it was sent with the request. */
-  private String mContinuationToken = null;
+  private String mName;
   /**
    * Returns the number of keys included in the response. The value is always less than or equal
    * to the mMaxKeys value.
    */
-  private int mKeyCount = 0;
-  /* The maximum number of keys returned in the response body. */
-  private int mMaxKeys = S3Constants.S3_DEFAULT_MAX_KEYS;
-  /**
-   * Specifies whether or not all of the results were returned. If the number of
```

Comment:
Can we keep this comment and the comment for continuation? (or an updated version of the comment)